app/actions/client.py

```diff
@@ -239,7 +239,7 @@ async def get_authentication_token(integration, auth, gql_client):
                 "attention_needed": True
             }
         )
-        await log_activity(
+        await log_action_activity(
```

Comment:
I hadn't seen this before, but in general I recommend keeping the Gundi log_activity calls out of the client code. In this way we can the client focused on the 3rd party API.